src/Symfony/Component/Form/Extension/Core/Type/FileType.php

```diff
@@ -93,7 +93,7 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $dataClass = function (Options $options) {
-            return $options['multiple'] ? null : 'Symfony\Component\HttpFoundation\File\File';
+            return $options['multiple'] || !class_exists('Symfony\Component\HttpFoundation\File\File') ? null : 'Symfony\Component\HttpFoundation\File\File';
```

Comment:
shouldn't the check be done a few lines below, when calling `setDefaults`?

Reply:
I agree with you, and fixed it.